packages/auth/src/shared/gnapErrors.ts

```diff
@@ -72,7 +81,11 @@ export async function gnapServerErrorMiddleware(
         'Received OpenAPI validation error when handling Open Payments GNAP request'
       )
 
-      ctx.throw(finalStatus, err.message)
+      ctx.status = finalStatus
+      ctx.body = {
+        error: { description: err.message }
+      }
```

Comment:
this is not a GNAP error per se, just a validation error, for which we just add the description